src/main/java/net/sf/jabref/logic/importer/fetcher/GoogleScholar.java

```diff
@@ -36,30 +63,113 @@
             return pdfLink;
         }
 
-        String url = String.format(SEARCH_URL,
-                URLEncoder.encode(entry.getField(FieldName.TITLE).orElse(null), StandardCharsets.UTF_8.name()));
-
-        Document doc = Jsoup.connect(url)
-                .userAgent("Mozilla/5.0 (Windows NT 5.1; rv:31.0) Gecko/20100101 Firefox/31.0") // don't identify as a crawler
-                .get();
-        // Check results for PDF link
-        // TODO: link always on first result or none?
-        for (int i = 0; i < NUM_RESULTS; i++) {
-            Elements link = doc.select(String.format("#gs_ggsW%s a", i));
-
-            if (link.first() != null) {
-                String s = link.first().attr("href");
-                // link present?
-                if (!"".equals(s)) {
-                    // TODO: check title inside pdf + length?
-                    // TODO: report error function needed?! query -> result
-                    LOGGER.info("Fulltext PDF found @ Google: " + s);
-                    pdfLink = Optional.of(new URL(s));
-                    break;
+        try {
+            URIBuilder uriBuilder = new URIBuilder(SEARCH_IN_TITLE_URL);
```

Comment:
This the "old" part of the fetcher responsible for fulltext downloads. The tests are currently not executed at travis as they led to errors if the IP got blocked by google (See https://github.com/JabRef/jabref/blob/master/src/test/java/net/sf/jabref/logic/importer/fetcher/GoogleScholarTest.java).
However, as my new tests (currently) are running through I'll check whether this still persists.